pkg/sql/schemachanger/scbuild/internal/scbuildstmt/alter_table_drop_column.go

```diff
@@ -39,6 +39,7 @@ func alterTableDropColumn(
 		return
 	}
 	checkColumnNotInaccessible(col, n)
+	fallBackIfZoneConfigExists(b, n, tbl.TableID)
```

Comment:
In the PR description, you mentioned that we will do this fall-back-if-zone-config-exists for ADD/DROP COLUMN, CREATE/DROP INDEX, and ALTER PRIMARY KEY. I only see you added this to `DROP COLUMN`, did you forget to add this to the other stmt, or did I miss something?

Reply:
nvm, I was looking at only the first commit. I saw it in the second commit